### PR TITLE
NTR pthlh-expressing interneuron for Basal Ganglia BICAN

### LINKED ISSUE #2568: NTR pthlh-expressing interneuron for Basal Ganglia BICAN
@Caroline-99 please review

**_pthlh_-expressing interneuron**
**definition:**
A GABAergic interneuron that has its soma in a striatum. This GABAergic interneuron is characterised by the expression of PTHLH and PVALB. 
PMID: **37292997**
PMID: [30134177](https://pubmed.ncbi.nlm.nih.gov/30134177)

**comments:**
According to spatial transcriptomic data in the human striatum, PTHLH-expressing interneurons are one of the most abundant population of interneurons in the striatum ([Leonardo D. Garma](https://www.nature.com/articles/s41467-024-50414-w#auth-Leonardo_D_-Garma-Aff1), et al., 2024). 
**database_cross_reference:**
PMID: **37292997**

**synonym:**
*PTHLH + /PVALB+ interneuron*
**database_cross_reference:**
PMID: **37292997**

**synonym:**
PTHLH/PVALB cell
**database_cross_reference:**
PMID: **37292997**

**synonym:**
_Pvalb_-expressing cell
**database_cross_reference:**
PMID: **37292997**

**parent term:** [GABAergic interneuron](http://purl.obolibrary.org/obo/CL_0011005)
**uberon term:** [striatum](http://purl.obolibrary.org/obo/UBERON_0002435)
**logical definition:**
subclass of GABAergic interneuron
soma location striatum
expressing PVALB and PTHLH (I will have to run the imports for this protein)

**new term request:**
 PTHLH MOXD1 expressing interneuron
**definition:**
A pthlh-expressing interneuron that expresses MOXD1. 
PMID: **35308118**

**parent term:**
Pthlh-expressing interneuron
**uberon term:**
striatum
**logical definition:**
subclass of pthlh-expressing interneuron 
expressing MOXD1 (I will have to run the imports for this protein)

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2568

### CODE REVIEW

**Comment by dosumis** [comment]
#gogoeditdiff

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
...
+AnnotationAssertion(terms:contributor obo:CL_4042023 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042023 "2024-09-17T10:20:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "PTHLH/PVALB cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "PTHLH +/PVALB+ interneuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "pvalb-expressing cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") rdfs:comment obo:CL_4042023 "According to spatial transcriptomic data in the human striatum, pthlh-expressing interneurons are one of the most abundant population of interneurons in the striatum (Leonardo D. Garma, et al., 2024).")
+AnnotationAssertion(rdfs:label obo:CL_4042023 "pthlh-expressing interneuron"@en)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32393]: Should the name be more specific?  e.g. include striatal

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (pthlh-expressing interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30134177") Annotation(oboInOwl:hasDbXref "PMID:37292997") obo:IAO_0000115 obo:CL_4042023 "A GABAergic interneuron that has its soma in a striatum. This GABAergic interneuron is characterised by the expression of PTHLH and PVALB.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32386]: This reads like it covers all GABAergic neurons in the striatum.  

proposed change to 

A PTHLH expressing GABAergic neuron with its soma in the striatum.  

IS PVALB really expressed in all?
image. Maybe mention spatial gradient of PVALB across these cells in the gloss?

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
...
+AnnotationAssertion(terms:contributor obo:CL_4042023 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042023 "2024-09-17T10:20:15Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") oboInOwl:hasRelatedSynonym obo:CL_4042023 "PTHLH/PVALB cell")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32389]: See notes on PVALB. Ditto for other synonyms.

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292997") rdfs:comment obo:CL_4042023 "According to spatial transcriptomic data in the human striatum, pthlh-expressing interneurons are one of the most abundant population of interneurons in the striatum (Leonardo D. Garma, et al., 2024).")
+AnnotationAssertion(rdfs:label obo:CL_4042023 "pthlh-expressing interneuron"@en)
+SubClassOf(obo:CL_4042023 obo:CL_0011005)
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002435))
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013433))
+SubClassOf(obo:CL_4042023 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013502))
+
+# Class: obo:CL_4042024 (PTHLH MOXD1 expressing interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35308118") obo:IAO_0000115 obo:CL_4042024 "A pthlh-expressing interneuron that expresses MOXD1.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32401]: Wrong ref? Looks like different neuron types covered. Don't seem pthlh or MOXD1 in this paper

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments in-line

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -32378,6 +32381,30 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (pthlh-expressing interneuron)
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:32384]: DANGER Clashing ID with edit in  https://github.com/obophenotype/cell-ontology/pull/2566/files !!!!

**Comment by hkir-dev** [comment]
I am testing #gogoeditdiff command here. I will delete this comment afterwards.

**Comment by AvolaAmg** [comment]
@dosumis addressed

**Comment by dosumis** [comment]
#gogoeditdiff

**Review by dosumis — COMMENTED** [review]
This is too broad:
[striatal pthlh-expressing interneuron](http://purl.obolibrary.org/obo/CL_4042023) [has_related_synonym](http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym) "pvalb-expressing cell"

Also better to have some detail on location in text def of MOXD1 neuron even if redundant.  Otherwise potentially too sparse for most users to appreciate what it refers too.

**Comment by AvolaAmg** [comment]
#gogoeditdiff

**Review by dosumis — APPROVED** [review]

**Pull request merged**